### PR TITLE
Joke Added

### PR DESCRIPTION
**Pull request by JustinTymeCoder**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -53,7 +53,7 @@
 <img src="forkit.gif" id="octocat" alt="" />
 
 <p>
-  Fork me? No, fork you!
+  Q:  What did the Spoon say to the fork?  A: Have a Knife Day!
```
Comment (JustinTymeCoder) [inline index.html:56]: A friendlier Joke

Comment (ctufindgood) [inline index.html:56]: cool

**Comment by ctufindgood** [comment]
wow

File: index.html (hunk 1 above)
Comment (simonewebdesign) [inline index.html:56]: lol

**Pull request closed without merging**